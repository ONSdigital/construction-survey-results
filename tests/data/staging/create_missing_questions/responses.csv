reference,period,questioncode,target,290_flag
1,202502,290,99999,false
1,202504,290,99999,false
1,202503,290,99999,false
1,202501,290,99999,false
1,202502,77,10277,false
1,202504,77,10477,false
1,202503,77,10377,false
1,202501,77,10177,false
1,202502,88,10288,false
1,202504,88,10488,false
1,202503,88,10388,false
1,202501,88,10188,false
1,202502,99,10299,false
1,202504,99,10499,false
1,202503,99,10399,false
1,202501,99,10199,false
2,202501,77,20177,false
2,202503,88,20388,false
2,202501,88,20188,false
2,202503,99,20399,false
2,202501,99,20199,false
2,202501,290,99999,false
2,202503,290,99999,false
3,202501,88,30188,false
3,202501,99,30199,false
3,202501,290,99999,false
5,202501,290,23213,true
