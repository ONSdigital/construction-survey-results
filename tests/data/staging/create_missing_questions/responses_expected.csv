reference,period,questioncode,target
1,202501,290,99999
1,202501,77,10177
1,202501,88,10188
1,202501,99,10199
1,202502,290,99999
1,202502,77,10277
1,202502,88,10288
1,202502,99,10299
1,202503,290,99999
1,202503,77,10377
1,202503,88,10388
1,202503,99,10399
1,202504,290,99999
1,202504,77,10477
1,202504,88,10488
1,202504,99,10499
2,202501,77,20177
2,202501,88,20188
2,202501,99,20199
2,202501,290,99999
2,202502,88,
2,202502,99,
2,202502,77,
2,202503,88,20388
2,202503,99,20399
2,202503,290,99999
2,202504,88,
2,202504,99,
3,202501,88,30188
3,202501,99,30199
3,202501,290,99999
3,202502,88,
3,202502,99,
3,202503,88,
3,202503,99,
3,202504,88,
3,202504,99,
4,202503,77,
4,202503,88,
4,202503,99,
